bin/citrea/provers/risc0/guest-bitcoin/src/bin/bitcoin_da.rs

```diff
@@ -24,9 +24,9 @@ fn report_bench_metrics(start_cycles: usize, end_cycles: usize) {
     serialized.extend(&size_bytes);
 
     // calculate the syscall name.
-    let cycle_string = String::from("cycle_metrics\0");
+    let name = c"cycle_metrics".as_ptr();
     let metrics_syscall_name =
-        risc0_zkvm_platform::syscall::SyscallName::from_bytes_with_nul(cycle_string.as_ptr());
+        unsafe { risc0_zkvm_platform::syscall::SyscallName::from_bytes_with_nul(name as _) };
```

Comment:
I am curious... why did you change this to unsafe though the previous version did not use the unsafe version?

Reply:
Sure, I can fix that. In 1.0.0-rc they didn't have a safe API, that's the code from that time.